gef.py

```diff
@@ -9261,15 +9235,15 @@ def pprint_dereferenced(addr: int, idx: int, base_offset: int = 0) -> str:
         base_address_color = gef.config["theme.dereference_base_address"]
         registers_color = gef.config["theme.dereference_register_value"]
 
-        sep = " {:s} ".format(RIGHT_ARROW)
+        sep = f" {RIGHT_ARROW} "
         memalign = gef.arch.ptrsize
 
         offset = idx * memalign
         current_address = align_address(addr + offset)
         addrs = dereference_from(current_address)
         l = ""
         addr_l = format_address(int(addrs[0], 16))
-        l += "{:s}{:s}{:+#07x}: {:{ma}s}".format(Color.colorify(addr_l, base_address_color),
+        l += "{}{}{:+#07x}: {:{ma}s}".format(Color.colorify(addr_l, base_address_color),
```

Comment:
Any reason you didn't f-string this one?

Reply:
Good eye, also the args on the next line aren't lined up.

Reply:
There are a few format-strings that I didn't convert to f-strings yet. When I didn't convert them it was mostly because I was concerned with worsening the readability. In this case the part I wasn't sure about is `{:ma}s}`. This line would turn into:

```py
l += f"{Color.colorify(addr_l, base_address_color)}{VERTICAL_LINE}{base_offset+offset:+#07x}: {sep.join(addrs[1:]):{memalign*2+2}s}"
```

I can include this and similar strings as well if you think that it improves the code quality overall (at least for the aspect of consistency)

Reply:
Yeah fair. We shouldn't be married to f strings if it makes things harder to read (but please fix the alignment of the args)